lib/backburner/index.js

```diff
@@ -28,7 +28,7 @@ export default function Backburner(queueNames, options) {
 
   var _this = this;
   this._boundClearItems = function() {
-    clearItems();
+    clearItems.apply(_this, arguments);
```

Comment:
Ya, I see that was done in https://github.com/ebryn/backburner.js/commit/66291441057e744c5f631eb94cf3273e2e7e55cb...